php/libraries/Candidate.class.inc

```diff
@@ -52,8 +52,9 @@ class Candidate
      *                        a candidate in the candidate table
      *
      * @return Candidate the Candidate object
+     * @static
      */
-    static function &singleton($candID)
+    static public function &singleton($candID)
```

Comment:
type hint : Candidate ?

Reply:
This was just from me trying to debug something and probably shouldn't be in this PR at all.

Should I remove it? It's still a small step forward if I leave it.

Reply:
No, leave it there. There is no harm in adding public. This function, if things goes as planned, should be removed in the next 10 . years :)